Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilder.cpp

```diff
@@ -425,6 +425,34 @@ namespace ScriptCanvasBuilder
 
         return AZ::Success(result);
     }
+
+    bool ReplaceAsset(ScriptCanvas::RuntimeDataOverrides& overrides, ScriptCanvas::RuntimeAssetPtr asset)
+    {
+        if (!asset.Get())
```

Comment:
I think you can just do 
`if (!asset)`
as it overloads bool to do the same thing